### ISSUE TITLE
Do we need lr scheduler?

### ISSUE THREAD

**Issue by Dforgeek** [yl4579/StyleTTS2#274]
In all repository, I didn't find any place where scheduler.step() was used in training. Should it be this way?

**Comment by martinambrus** [comment]
I've found step() function of scheduler being used on these lines: https://github.com/yl4579/StyleTTS2/blob/main/optimizers.py#L43-L45

Also, the lr_scheduler is being used here: https://github.com/yl4579/StyleTTS2/blob/main/optimizers.py#L54

All of that is used in imports of all training scripts as ```from optimizers import build_optimizer```.

**Comment by Dforgeek** [comment]
It is true that there's single place where .step() is being called, иut the method .scheduler() in which .step() is called has never been used for any of the training scripts. So I think it's not used.

**Comment by ethan-digi** [comment]
I can confirm, at least for train_finetune.py, that the lrs are not adjusted for any of the models
